src/goto-instrument/cover_basic_blocks.cpp

```diff
@@ -234,3 +166,47 @@ static void update_covered_lines(cover_basic_blockst::block_infot &block_info)
   std::string covered_lines = format_number_range(line_list);
   block_info.source_location.set_basic_block_covered_lines(covered_lines);
 }
+
+cover_basic_blocks_javat::cover_basic_blocks_javat(
+  const goto_programt &_goto_program)
+{
+  forall_goto_program_instructions(it, _goto_program)
+  {
+    const auto &bytecode_index = it->source_location.get_java_bytecode_index();
+    if(!index_to_block.count(bytecode_index))
+    {
+      block_infos.push_back(it);
+      index_to_block.insert(
+        std::make_pair(bytecode_index, block_infos.size() - 1));
```

Comment:
You write it as `insert({ bytecode_index, block_infos.size() - 1})` or `emplace(bytecode_index, block_infos.size() - 1)` to avoid `std::make_pair`.